composables/zksync/deposit/useTransaction.ts

```diff
@@ -44,6 +44,7 @@ export default (getL1Signer: () => Promise<L1Signer | undefined>) => {
         token: transaction.tokenAddress,
         amount: transaction.amount,
         l2GasLimit: fee.l2GasLimit,
+        approveERC20: true,
```

Comment:
It's supposed to firstly ask to approve ERC20 through the UI. So this isn't needed after that. If it doesn't ask to approve before making a deposit then that's an issue.
Probably just need to change this line https://github.com/matter-labs/dapp-portal/blob/8471411eebcae55dba940bee492ba6cb74a167d3/views/transactions/Deposit.vue#L489-L490

Reply:
It did the trick